test/adapter/json_api/relationship_test.rb

```diff
@@ -150,9 +150,23 @@ def test_relationship_with_everything
 
         private
 
-        def test_relationship(expected, params = {})
+        def test_relationship(expected, test_options = {})
           parent_serializer = AuthorSerializer.new(@author)
-          relationship = Relationship.new(parent_serializer, @serializer, nil, params)
+
+          serializable_resource_options = {} # adapter.instance_options
+
+          meta = test_options.delete(:meta)
+          options = test_options.delete(:options)
+          links = test_options.delete(:links)
+          association_serializer = @serializer
+          if association_serializer && association_serializer.object
+            association_name = association_serializer.json_key.intern
```

Comment:
Oh, that's hipster for to_sym. I should change

B mobile phone

> On Aug 17, 2016, at 7:20 AM, L. Preston Sego III <notifications@github.com> wrote:
> 
> In test/adapter/json_api/relationship_test.rb:
> 
> >            parent_serializer = AuthorSerializer.new(@author)
> > -          relationship = Relationship.new(parent_serializer, @serializer, nil, params)
> > +
> > +          serializable_resource_options = {} # adapter.instance_options
> > +
> > +          meta = test_options.delete(:meta)
> > +          options = test_options.delete(:options)
> > +          links = test_options.delete(:links)
> > +          association_serializer = @serializer
> > +          if association_serializer && association_serializer.object
> > +            association_name = association_serializer.json_key.intern
> intern? I've not seen that before
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>